codeforamerica/vita-min
# re-encrypts demo credentials with new key

## Link to pivotal/JIRA issue
n/a

## Is PM acceptance required? (delete one)
No - merge after code review approval

## What was done?
- Re-encrypted the demo env creds w/ a new encryption key
- Added the updated key to lastpass
- After merge, will need to purge the old key from aptible environment.

## How to test?
- Pull the branch
- Copy the new encryption key from lastpass into `config/credentials/demo.key`
- Ensure you can decrypt the file
